Dockerfile

```diff
@@ -1,8 +1,30 @@
-FROM vmearl/sideboard:latest
+FROM ghcr.io/magfest/sideboard:main
 MAINTAINER RAMS Project "code@magfest.org"
 LABEL version.rams-core ="0.1"
 
-# add our code
-COPY . plugins/uber/
-# go ahead and install base dependencies
+# install ghostscript and gettext-base
+RUN apt-get update && apt-get install -y ghostscript gettext-base vim && rm -rf /var/lib/apt/lists/*
+
+ADD . plugins/uber/
+
 RUN /app/env/bin/paver install_deps
+RUN /app/env/bin/python3 -m pip install SQLAlchemy==1.3.0
+
+ADD uber-development.ini.template ./uber-development.ini.template
+ADD sideboard-development.ini.template ./sideboard-development.ini.template
+ADD uber-wrapper.sh /usr/local/bin/
+
+ENV HOST=0.0.0.0
```

Comment:
Let's also add a comment that this is just a list of defaults/possible env variables and a recommendation to run docker with an env file.